src/jitlayers.h

```diff
@@ -247,7 +247,8 @@ struct jl_codegen_params_t {
     std::unique_ptr<Module> _shared_module;
     inline Module &shared_module();
     // inputs
-    size_t world = 0;
+    size_t min_world = 0;
+    size_t max_world = -1;
```

Comment:
I wonder if eventually we should try to drop these 2 fields entirely, to permit codegen emitting code for multiple world ranges at once. We already do that in many places, so it is a bit sketchy currently that we have to temporarily mutate this field to emit a `cfunction` call and to run the aotcompile code.